lib/stripe/apple_pay_domain.rb

```diff
@@ -0,0 +1,12 @@
+module Stripe
+  # Domains registered for Apple Pay on the Web
+  class ApplePayDomain < APIResource
+    def self.resource_url
+      '/v1/apple_pay/domains'
+    end
+
+    extend Stripe::APIOperations::Create
+    include Stripe::APIOperations::Delete
+    extend Stripe::APIOperations::List
```

Comment:
Minor nitpick, but mind hoisting all the `extend` and `include` directives to the top of the class? This is more in line with what we have elsewhere.